src/chrome/content/rules/Frugalware.org.xml

```diff
@@ -0,0 +1,38 @@
+<!--
+	Invalid certficate:
+		darcs.frugalware.org
+		dors.frugalware.org
+		git.frugalware.org
+		lists.frugalware.org
+		packages.frugalware.org
+		planet.frugalware.org
+		wiki.frugalware.org
+
+	Private subdomain:
+		webmail.frugalware.org
```

Comment:
The argument against adding private subdomains was that we don't know if the rewrite rule will break what's behind the login page: maybe there is active mixed content.